src/bundler/compiler.ts

```diff
@@ -21,10 +21,14 @@ export function compile(env: tsvfs.VirtualTypeScriptEnvironment): string {
   const src =
     env.languageService.getEmitOutput(appEntrypointFilename).outputFiles[0]
       ?.text ?? ''
-  // Adapt bundle CommonJS output to format expected by runtime-lite.
+  // Adapt bundle CommonJS output to format expected by our runtimes.
```

Comment:
thanks for this beefy comment and PR message. I didn't understand why var specifically was needed--it's hoisted outside the block but why? some issue with @devvit/public-api? I know the code is cryptic by nature but I see it and immediately think it's safe to rewrite `"use strict"; module.exports ??= {}; const {exports} = module; }\n'. anything you can do to make this clearer would be appreciated.

the other thing I was thinking is that changing app code is hard but changing runtime-lite or the compute @devvit/public-api bundle is possible if wanted.

Reply:
When we load a cjs file into a module in node, the contents are basically wrapped like this:

```
(function(exports, require, module, __filename, __dirname) {
// Module code actually lives in here
}); 
```

Reply:
`beefiness++`

added more comment, hopefully that explains better

we could change runtime-lite and drop this `src.replace` call entirely, though that's a little outside the scope I wanted to dig into. we can't fix this on the compute side though without some ugly Node hackery (i.e. monkey-patching the contents of `require('module').wrapper`)